diff -Nur katapult-0.2/katapult/Makefile.am katapult-0.2.new/katapult/Makefile.am
--- katapult-0.2/katapult/Makefile.am	2005-02-27 14:53:00.000000000 +0100
+++ katapult-0.2.new/katapult/Makefile.am	2005-06-13 15:27:09.107068120 +0200
@@ -21,5 +21,4 @@
 katapult_LDADD = $(LIB_KDEUI) $(LIB_KIO) $(top_builddir)/common/libkatapultcatalog.la $(top_builddir)/common/libkatapultdisplay.la
 
 # this is where the desktop file will go 
-shelldesktopdir   = $(kde_appsdir)/Utilities
-shelldesktop_DATA = katapult.desktop
+xdg_apps_DATA = katapult.desktop
